ui/apps/platform/src/Containers/VulnMgmt/List/ImageComponents/ListImageComponents.utils.test.js

```diff
@@ -6,6 +6,13 @@ import { WorkflowState } from 'utils/WorkflowState';
 
 import { getFilteredComponentColumns } from './ListImageComponents.utils';
 
+function mockIsFeatureFlagEnabled(flag) {
```

Comment:
nit: Can we just call it `isFeatureFlagEnabled` if it's just used within this file